Sources/GDBRemote/DebugSessionImpl.cpp

```diff
@@ -458,6 +458,19 @@ ErrorCode DebugSessionImplBase::onQuerySharedLibrariesInfoAddress(
 #endif
 }
 
+ErrorCode DebugSessionImplBase::onQueryFileLoadAddress(
+    Session &session, std::string const &file_path, Address &address) {
+  if (_process == nullptr) {
+    return kErrorProcessNotFound;
+  }
+
+#if defined(OS_WIN32)
```

Comment:
Instead of doing this, you should put `getFileLoadAddress` in `Process.h` and have it return `kErrorUnsupported` on non-windows platforms. That way, this function can stay platform-agnostic, and we can avoid introducing differences in the interfaces of each OS's `Process` class.